lib/src/json_differ.dart

```diff
@@ -61,68 +82,79 @@ class JsonDiffer {
   /// tracking all additions, deletions, and changes. Please see the
   /// documentation for [DiffNode] to understand how to access the differences
   /// found between the two JSON Strings.
-  DiffNode diff() => _diffObjects(leftJson, rightJson)
-      ..prune();
+  DiffNode diff() => _diffObjects(leftJson, rightJson, '')..prune();
 
-  DiffNode _diffObjects(Map<String,Object> left, Map<String,Object> right) {
-    DiffNode node = new DiffNode();
+  DiffNode _diffObjects(
+      Map<String, Object> left, Map<String, Object> right, String path) {
+    final node = DiffNode();
     _keepMetadata(node, left, right);
     left.forEach((String key, Object leftValue) {
-      if (ignored.contains(key))
+      if (debug) {
+        print('Diffing object "$key" at path "$path"');
```

Comment:
I don't think these debug statements provide any value to the library. It seems like these should just be added in appropriate places according to whatever bug someone is trying to debug.

Reply:
You're right, it's basically only useful when developing this package.